data/csp/azure/settings/suma-server/ondemand/sle15/packages.yaml

```diff
@@ -0,0 +1,4 @@
+packages:
+  _namespace_azure_suma_ondemand:
+    package:
+      - python3-csp-billing-adapter-azure
```

Comment:
Don't ask me why, but the correct package name is `python3-csp-billing-adapter-microsoft`